src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -3230,6 +3231,33 @@ private BoundExpression BindArgumentExpression(BindingDiagnosticBag diagnostics,
                 var kind = result.ConversionForArg(arg);
                 BoundExpression argument = arguments[arg];
 
+                if (Compilation.IsFeatureEnabled(MessageID.IDS_FeatureRefReadonlyParameters) &&
```

Comment:
>Compilation.IsFeatureEnabled(MessageID.IDS_FeatureRefReadonlyParameters) [](http://example.com/codeflow?start=20&length=72)

Are we testing `ref`/`in` mismatch for `C#11`?

Reply:
Yes, in `OverloadResolutionTests.PassingArgumentsToInParameters_RefKind_Ref` (it's a pre-existing test, hence it's in a different file).